api/query.proto

```diff
@@ -75,6 +75,16 @@ message QueryTicketIdsResponse {
   repeated string ids = 1;
 }
 
+message QueryBackfillsRequest {
```

Comment:
beta feature warning

Reply:
Be consistent with request/response types:  Either they all get beta feature request warnings, or none of them do.  I don't have an opinion which way to go as long as they're consistent.

Reply:
Added beta feature warnings in all places but for some reason they are not showing the latest changes